manifests/grafana-dashboardDefinitions.yaml

```diff
@@ -11020,6 +11020,32 @@ items:
                       "allValue": null,
                       "current": {
 
+                      },
+                      "datasource": "$datasource",
+                      "hide": 2,
+                      "includeAll": false,
+                      "label": "cluster",
+                      "multi": false,
+                      "name": "cluster",
+                      "options": [
+
+                      ],
+                      "query": "label_values(kube_pod_info, cluster)",
+                      "refresh": 2,
+                      "regex": "",
+                      "sort": 0,
+                      "tagValuesQuery": "",
+                      "tags": [
+
+                      ],
+                      "tagsQuery": "",
+                      "type": "query",
+                      "useTags": false
+                  },
+                  {
+                      "allValue": null,
+                      "current": {
+
```

Comment:
all the changes in this file are generated after make `generate-in-docker`, not specific my to changes.

Reply:
this seems relatable to #269